usr/share/rear/backup/DUPLICITY/default/500_make_duplicity_backup.sh

```diff
@@ -21,41 +21,37 @@ if [ "$BACKUP_PROG" = "duplicity" ] ; then
     # by falk hoeppner
 
     if [ -n "$BACKUP_DUPLICITY_ASK_PASSPHRASE" ]; then
-        LogPrint "Warning !
-    BACKUP_DUPLICITY_ASK_PASSPHRASE set, The Passphrase needs to be provided Interactively on Restore."
+        LogPrint "Warning: BACKUP_DUPLICITY_ASK_PASSPHRASE set, The Passphrase needs to be provided Interactively on Restore."
```

Comment:
I suggest to remove the word `Warning: ` because
https://blog.schlomo.schapiro.org/2015/04/warning-is-waste-of-my-time.html
(in this case the user must have explicityl set BACKUP_DUPLICITY_ASK_PASSPHRASE
because by default it is not set so that the user should know what that means)
and change the text to
<pre>
LogPrint "Duplicity passphrase must be provided interactively on restore (BACKUP_DUPLICITY_ASK_PASSPHRASE is set)"
</pre>
i.e. show first the actual message and then the reason in parenthesis.

FYI:
`LogPrint` messages are only shown when rear runs in vervose mode
which is not the default for `rear mkbackup` but the default for `rear recover`.
If you like to show a message in any case to the user use
the `LogUserOutput` function and if you need to show some kind of error messages
you can also use the `LogPrintError` function, see the function definitions in
lib/_input-output-functions.sh for the subtle differences between those functions.